<ui version="4.0" >
 <class>AddChemicalDialog</class>
 <widget class="QDialog" name="AddChemicalDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add a Chemical</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>240</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Chemical Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_koc" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>90</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>150</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Diffusivity in water [cm2/s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_hlaw" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>60</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>180</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Atmospheric concentration [g/cm3]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_diffWater" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>150</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>90</y>
     <width>79</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Koc [g-i/g-oc]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_diffAir" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>120</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_atmoConc" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>180</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_name" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>60</y>
     <width>257</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Henrys law constant [cm3-water/cm3-vapor]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>120</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Diffusivity in air [cm2/s]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddChemicalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddChemicalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
